docs/conf_sso_test_msscolab.rst

```diff
@@ -14,63 +14,72 @@ To set up a local identity provider with the mscolab server, you'll first need t
     3. Enable USE_SAML2
     4. Generate Metadata Files
     5. Start the Identity Provider
-    6. Restart the mscolab Server
+    6. Start the mscolab Server
     7. Test the Single Sign-On (SSO) Process
 
 
-Initial Steps
--------------
+1. Initial Steps
+----------------
 Before getting started, you should correctly activate the environments, set the correct Python path, and be in the correct directory (`$ cd MSS`), as explained in the mss instructions : https://open-mss.github.io/develop/Setup-Instructions
 
 
 
-Generate Keys, Certificates, and backend_saml files
----------------------------------------------------
+2. Generate Keys, Certificates, and backend_saml files
+------------------------------------------------------
+
+This involves generating both `.key` files and `.crt` files for both the Identity provider and mscolab server and `backend_saml.yaml` file. 
+
+Before running the command make sure to set `USE_SAML2 = False` in your `mscolab_settings.py` file,  You can accomplish this by following these steps:
 
-This involves generating both .key files and .crt files for both the Identity provider and mscolab server and backend_saml.yaml file. 
+- Add to the `PYTHONPATH` where your `mscolab_settings.py`.
+- Add `USE_SAML2 = False` in your `mscolab_settings.py` file.
 
-Before running the command make sure to set
-`USE_SAML2 = False` 
-In some cases, if you set `USE_SAML2 = True` without certificates, this will not execute. So, make sure to set `USE_SAML2 = False` before executing the command.
+.. note::
+    If you set `USE_SAML2 = True` without keys and certificates, this will not execute. So, make sure to set `USE_SAML2 = False` before executing the command.
 
-Then you can generate files, simply by running,
+If everything is correctly set, you can generate keys and certificates simply by running
 
 `$ python mslib/mscolab/mscolab.py sso_conf --init_sso_crts`
 
```

Comment:
the documentation is for users, keep in mind, they will use the "entry points" in this case 

mscolab sso_conf --init_sso_crts